Makefile

```diff
@@ -1,6 +1,8 @@
 .PHONY: build
-build: 
+build:
 	./before-commit.sh ci
+	# TODO: we should provide separate Prow jobs for provisioning and installation packages after the repository will be adjusted
```

Comment:
I am not sure if this is the right approach. This repository is a library and thus generates no binaries, so when releaseing (tagging) one module, we inevitably also release all others as well.
Based on that, I think we should have only one CI pipeline for the complete repository.

Reply:
That is a fair point. I only wonder if we will provide some more complex and long-running tests those for provisioning would not need to be run if changes were made to the installation, but I guess we can do a separate job for that letter.
I will remove the TODO then.

Reply:
True, in that case the CI job could be smart and run only the tests of the modules with changes on them.